addons/mail/static/src/scss/activity_view.scss

```diff
@@ -126,4 +126,7 @@
             }
         }
     }
+    th:not(.o_activity_type_cell) {
```

Comment:
`th:first-of-type` seems more appropriate IMO

Reply:
Sure, here it is.